src/names/applications.cpp

```diff
@@ -175,3 +177,10 @@ DescFromName (const valtype& name, NameNamespace ns)
         }
     }
 }
+
+bool
+IsValidJSON (const std::string& text){
+    UniValue v;
+
+    return v.read(text) || text.empty();
```

Comment:
I would change this to `test.empty() || v.read(text)`, since checking for the empty string is a simpler operation than JSON-parsing (even though JSON parsing of the empty string is probably also easy), and thus the shortcut makes more sense.